docs/quickstart/quickstart_chains/ethereum-uniswap.md

```diff
@@ -0,0 +1,264 @@
+# Ethereum Quick Start - Uniswap (Medium)
+
+## Goals
+
+This project serves as an excellent foundation for initiating your Ethereum SubQuery project. It meticulously indexes various Uniswap entities, including Swaps, Pool operations, Price ticks, Positions, and additionally computes statistical metrics.
```

Comment:
Minor thing. I'm not sure if this is the best opening line and if this project is actually this great foundation to initialise eth SQ project.  I think I would rather start with what the project indexes, what are the goals.